components/yaml.rst

```diff
@@ -224,20 +224,20 @@ Instead of encoding as ``null`` you can choose to throw an exception if an inval
 type is encountered in either the dumper or parser as follows::
 
     // throw an exception if a resource or object is encountered
-    Yaml::dump($data, 2, 4, true);
+    Yaml::dump($data, 2, 4, Yaml::DUMP_EXCEPTION_ON_INVALID_TYPE);
 
     // throw an exception if an encoded object is found in the YAML string
-    Yaml::parse($yaml, true);
+    Yaml::parse($yaml, Yaml::PARSE_EXCEPTION_ON_INVALID_TYPE);
 
 However, you can activate object support using the next argument::
```

Comment:
this is no longer true, it's not the next statement. Can you please rewrite this sentence?

status: needs work